lisa/features/startstop.py

```diff
@@ -16,29 +18,29 @@ def can_disable(cls) -> bool:
         # no reason to disable it, it can not be used
         return False
 
-    def _stop(self, wait: bool = True) -> None:
+    def _stop(self, wait: bool = True, **kwargs: Any) -> None:
```

Comment:
Hmmm, it doesn't need to add it on each level. For the _stop, just use the real arguments as last time, so it's easy to know which parameter supported.